ci-operator/step-registry/baremetal/lab/pre/dns/baremetal-lab-pre-dns-commands.sh

```diff
@@ -1,2 +1,87 @@
-#!/bin/sh
-echo Hello world
+#!/bin/bash
+
+# This script modifies the following files in the auxiliary host:
+# - /opt/bind9_zones/zone
+# - /opt/bind9_zones/internal_zone.rev
+
+if [ -n "${LOCAL_TEST}" ]; then
+  # Setting LOCAL_TEST to any value will allow testing this script with default values against the ARM64 bastion @ RDU2
+  # shellcheck disable=SC2155
+  export NAMESPACE=test-ci-op AUX_HOST=openshift-qe-bastion.arm.eng.rdu2.redhat.com \
+      SHARED_DIR=${SHARED_DIR:-$(mktemp -d)} CLUSTER_PROFILE_DIR=~/.ssh INTERNAL_NET_IP=192.168.90.1
+fi
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ -z "${AUX_HOST}" ]; then
+    echo "AUX_HOST is not filled. Failing."
+    exit 1
+fi
+
+SSHOPTS=(-o 'ConnectTimeout=5'
+  -o 'StrictHostKeyChecking=no'
+  -o 'UserKnownHostsFile=/dev/null'
+  -o 'ServerAliveInterval=90'
+  -o LogLevel=ERROR
+  -i "${CLUSTER_PROFILE_DIR}/ssh-key")
+
+BASE_DOMAIN=$(<"${CLUSTER_PROFILE_DIR}/base_domain")
+
+# shellcheck disable=SC1090
+. <(yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")' < "${SHARED_DIR}"/external_vips.yaml)
+# shellcheck disable=SC2154
+if [ ${#api_vip} -eq 0 ] || [ ${#ingress_vip} -eq 0 ]; then
+  echo "Unable to parse VIPs"
+  exit 1
+fi
+
+DNS_FORWARD="#DO NOT EDIT; BEGIN $NAMESPACE
+api.${NAMESPACE} IN A ${api_vip}
```

Comment:
A note, we may also want to consider `AAAA` for ipv6 single stack and `A`,`AAAA` for dualstack clusters.

Reply:
Merging to proceed ahead with the other steps @jhou1 and get a first test running asap. I'll follow up with the ipv6 support in the next days.